websites/api-docs-smoke-test/source-raml/test/api.raml

```diff
@@ -53,30 +53,37 @@ uses:
             scopes: ["manage_test:{projectKey}"]
       body:
         type: objects.ObjectTestTypeDraft
+        example: !include examples/object-test-type-draft.json
       responses:
         201:
           description: 201 Created response. This is the success response when creating something.
           body:
             type: objects.ObjectTestType
+            example: !include examples/object-test-type.json
     get:
       displayName: Get all resources
       description: Query/retrieve (GET) a specific resource or resource(s). This should return a paged query result of _blah_.
       queryParameters:
         where:
           type: string
           required: false
+          example: 6 in arrayOfIntExample
         sort:
           type: string
           required: false
+          example: DESC
         expand:
           type: string
           required: false
+          example: arrayOfObjectType[*]
         limit:
           type: number
           required: false
+          example: 50
         offset:
           type: number
           required: false
+          example: 5
         collection:
           type: objects.ObjectTestType
           required: false
```

Comment:
> For testing an implementation, it would be good to have a test case where one optional query parameter has no example set. maybe this one?

This has no example.